frontend/model/contracts/group.js

```diff
@@ -703,13 +721,28 @@ sbp('chelonia/defineContract', {
           { username: meta.username, dateLeft: meta.createdDate },
           { meta, state, getters }
         )
+
+        meta.isRemoveOurselves = true
         sbp('gi.contracts/group/pushSideEffect', contractID,
           ['gi.contracts/group/removeMember/sideEffect', {
             meta,
             data: { member: meta.username, reason: data.reason || '' },
             contractID
           }]
         )
+      },
+      sideEffect ({ meta, contractID }, { state }) {
+        const { loggedIn } = sbp('state/vuex/state')
+        const { profiles = {} } = state
+        const myProfile = profiles[loggedIn.username] || null
+
+        if (loggedIn.username !== meta.username &&
+          isContractYoungerThanUser(meta, myProfile)) {
```

Comment:
A user who has joined a group later than other existing members doesn't need to get notified of the existing members. For example, a `user3` who has just joined the group doesn't need to get a notification for `user2` who joined the group (let's say)2 days ago.